src/select.tsx

```diff
@@ -52,7 +52,8 @@ const useStyles = createUseStyles((theme) => ({
         left: 12 + 1, // 12 for select paddingLeft and +1 for border
         right: 48 + 1, // 48 for icon and +1 for border
         overflow: 'hidden',
```

Comment:
I found problems using big sur and desktop views and I think is caused by https://jira.tid.es/browse/WEB-251 (that I don't know why @pladaria close this ticket xD)
https://tinyurl.com/yc4l7pe3

![image](https://user-images.githubusercontent.com/6722153/155387458-20ac7350-6525-40b6-ac3f-1feb3d6dbee4.png)

Reply:
I closed it because this seems to be something specific to Safari rendering, we can't do much there. 

I reopened it  but I think we can't do much there